app/views/templates/formatters/tabs/inactiveTab.scala.html

```diff
@@ -0,0 +1,24 @@
+@*
```

Comment:
Could you add some small unit tests for these new templates please? You should then be able to dumb-down the CompletedReturnsViewSpec